api_v3/lib/syndication/KalturaSyndicationFeedRenderer.php

```diff
@@ -167,6 +169,16 @@ public function __construct($feedId, $feedProcessingKey = null, $ks = null, $sta
 	
 		if($this->addLinkForNextIteration)
 		{
+			$entryFilter = $this->syndicationFeed->entryFilter;
+			foreach(self::$relativeTimeFieldNames as $relativeTimeFieldName)
```

Comment:
Can't we do this in `entryFilter` so we could re-use `entryFilter::$relative_time_fields`?